src/Console/Commands/ShortenCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Console\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+class ShortenCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'shorten';
```

Comment:
I would personally prefer to name it url:shorten because I plan on adding URL expansion in the future, which may lead to the command url:expand being added